### DISCUSSION TITLE
Harvester system showing in Full Node but nothing in logs (set to debug)

### DISCUSSION THREAD

**Discussion by Mar-85** [Chia-Network/chia-blockchain#6429] [Support]
Following the wiki I've set up the harvester and a full node. 

Starting up the harvester with "Chia.exe star harvester" i see it starting the daemon and also starting the harvester. I then see the harvester in the "Your Harvester Network" section, Last Attempted Proof is showing the group of plots from the harvester, and ChiaDog (on my harvester) is telling me it sees the plots and is harvesting. 

But when I look at the logs according to the wiki I should see certain lines but I dont see anything. Im on 1.1.6 and the log is set to DEBUG. 

edit: i see that things have changed from previous releases, is anyone able to confirm what the new harvester log lines are on the node and the harvester?

**Comment by BrandtH22** [comment]
@Mar-85 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
